test/fixtures/fake-app/resources/assets/extract/app.js

```diff
@@ -4,6 +4,6 @@ import 'core-js';
 new Vue({
     components: {
         VueSplit: () =>
-            import(/* webpackChunkName: 'js/split' */ './VueSplit.vue')
```

Comment:
@JeffreyWay  How do you want to handle this? I opted in the new code to check for absolute paths and not touch it if there is one but should we also detect the relative path if it starts with the same name?
Seems like it could be brittle (but still doable).

Reply:
Hmm, will need to think about that. For now, my instinct is no. Will come back to this after I play around with it a bit.